go/cmd/dolt/commands/sqlserver/server.go

```diff
@@ -35,6 +36,8 @@ import (
 	"github.com/dolthub/dolt/go/libraries/doltcore/sqlserver"
 )
 
+const SockAddr = "/tmp/mysql.sock"
```

Comment:
Following up on our chat yesterday... it would be ideal to mimic MySQL's behavior and put this in the expected location for different unix distros. That might require looking at the mysql source code to reverse engineer what they're doing.

Reply:
Where do we actually create this file? I didn't see that in here or in the GMS PR, but I could just be missing it.

Reply:
Ah I was creating this file from dolt, but found out that the `net` package that we use already handles creating this file when server starts and deleting it after server stops, https://github.com/dolthub/go-mysql-server/blob/main/server/listener.go#L28